### PR TITLE
Populate registry for vocabulary.json

### PR DESCRIPTION
**Pull request by santhosh-tekuri**
reqd http://localhost:1234/draft2020-12/metaschema-no-validation.json

### CODE REVIEW

File: bowtie/_cli.py [hunk 1]
```diff
@@ -471,7 +471,7 @@ def sequenced(cases, reporter):
 
 def suite_cases_from(files, remotes):
     for file in files:
-        if file.name == "refRemote.json" or file.name == "dynamicRef.json":
+        if file.name == "refRemote.json" or file.name == "dynamicRef.json" or file.name == "vocabulary.json":
```
Comment (Julian) [inline bowtie/_cli.py:474]: :/ -- it now probably makes sense to do this as:

```suggestion
        if file.stem in {"refRemote", "dynamicRef", "vocabulary"}:
```

Comment (santhosh-tekuri) [reply bowtie/_cli.py:474]: I guess it should be array literal not map literal

Comment (Julian) [reply bowtie/_cli.py]: That's a set literal (with no colons).

Comment (Julian) [reply bowtie/_cli.py]: (CI is failing because the line is now too long -- `.stem` will trim off the `.json`, which probably fits).

**Pull request merged**